### PR TITLE
Guidelines on Migration from Alpine to Debian

### PR DESCRIPTION
**Pull request by paolaperaza**
@vparekh94 I drafted some guidelines to help users migration from Alpine to Debian, as Astronomer Certified no longer supports Alpine-based Docker images beginning with AC 1.10.14.

In this PR, I also:
- Renamed this doc from "Manage Airflow Versions" to "Upgrade Airflow"
- Moved headers/section titles up for a cleaner format
- Removed the description of Alpine v Debian to the new section on the topic to keep things simple to start

Can you give this a look and let me know what you think? This doc is generally a bit wordy/packed with information, but hoping it's easy to find/navigate for users nonetheless.

### CODE REVIEW

File: cloud/stable/04_customize-airflow/02_manage-airflow-versions.md [hunk 1]
```diff
@@ -203,14 +173,112 @@ Once there, you should see your correct Airflow version listed.
...
+
+Debian Buster has many common packages installed by default, which means that you should be able to remove some dependencies.
+
+If you have any Python packages installed primarily *because* they're native to another library  (e.g. `pandas`, `numpy`, `pyarrow`, `scipy`, `sci-kit learn`), we recommend that you remove those additional packages from your `requirements.txt` or `packages.txt` files and see if your image builds successfully.
+
+If you test a Debian-based image and encounter an error, you can always add packages back as needed.
+
+### Step 2. Rename Existing Packages.
+
+For the dependencies you *do* have installed, a primary concern in migrating from Alpine to Debian is that Python and OS-level packages may be named differently.
```
Comment (astrorfox) [inline cloud/stable/04_customize-airflow/02_manage-airflow-versions.md:213]: @paolaperaza are there a few commons ones we can include as reference to make this less daunting?

Comment (paolaperaza) [reply cloud/stable/04_customize-airflow/02_manage-airflow-versions.md:213]: @astrorfox Yea, good call - I'll add some examples here.

Comment (vparekh94) [reply cloud/stable/04_customize-airflow/02_manage-airflow-versions.md:213]: hm, if something is misnamed here, the image will just fail to build. So maybe we just warn the user about that? Can't think of any "common" package names that change between the two

File: cloud/stable/04_customize-airflow/02_manage-airflow-versions.md [hunk 2]
```diff
@@ -203,14 +173,112 @@ Once there, you should see your correct Airflow version listed.
...
+For all available images, refer to the matrix above or the [Astronomer Docker Registry](https://quay.io/repository/astronomer/ap-airflow?tab=tags).
+
+### Step 4. Test your Image Locally.
+
+Now, test your new change locally via the Astronomer CLI by running:
+
+1. `$ astro dev stop`, then
+2. `$ astro dev start`
+
+If your image does not build successfully, it's likely you're missing additional dependencies, Add them as needed.
```
Comment (vparekh94) [inline cloud/stable/04_customize-airflow/02_manage-airflow-versions.md:244]: still having a naming mismatch or missing additional dependencies

File: cloud/stable/04_customize-airflow/02_manage-airflow-versions.md [hunk 3]
```diff
@@ -203,14 +173,112 @@ Once there, you should see your correct Airflow version listed.
...
+
+### Step 5. Push to Astronomer.
+
+If your image *does* build successfully, you're ready to push it to your Airflow Deployment Astronomer.
+
+To do so, simply run:
+
+```bash
+ $ astro deploy
+ ```
```
Comment (vparekh94) [inline cloud/stable/04_customize-airflow/02_manage-airflow-versions.md:256]: Add something like "or push your changes up to be picked up by CI/CD

**Review by vparekh94 — APPROVED** [review]

File: cloud/stable/04_customize-airflow/02_manage-airflow-versions.md (hunk 2 above)
Comment (paolaperaza) [reply cloud/stable/04_customize-airflow/02_manage-airflow-versions.md:244]: Word - good call to specify this. Added!

File: cloud/stable/04_customize-airflow/02_manage-airflow-versions.md (hunk 3 above)
Comment (paolaperaza) [reply cloud/stable/04_customize-airflow/02_manage-airflow-versions.md:256]: Good call - added!

File: cloud/stable/04_customize-airflow/02_manage-airflow-versions.md (hunk 1 above)
Comment (paolaperaza) [reply cloud/stable/04_customize-airflow/02_manage-airflow-versions.md:215]: Ok, that's fair V - adjusted accordingly
